File: Python Programming/problems on string manipulation/problem124.py
'''
124. Write a program which accept string from user and reverse each word in place. 
		
		Input :   “Marvellous Multi OS” 
		Output :   “suollevraM itluM SO”

		Author : Milind Nalawade
	
		Date : 05/08/2021
	
		Language : Python 3.9.0

'''

#====
#Logic
#====
def Output(str):
	list=[]
	print("\n Output : ",end="")
	for i in range(len(str)):
		list.append(str[i])
		if str[i]==' ' or i==len(str)-1:
			for j in reversed(list):
				print(j,end="")
			print(" ",end="")	
			list.clear()
	print()		
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter your string : ")
	Output(str)
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on strings maipulation/problem124.c
/*
124. Write a program which accept string from user and reverse each word in place. 
		
		Input :   “Marvellous Multi OS” 
		Output :   “suollevraM itluM SO”

		Author : Milind Nalawade
	
		Date : 05/08/2021
	
		Language : C programming

*/
#include<stdio.h>
#include<string.h> 
void Reverse(char arr[])
{
	char brr[20];
	int i=0,j=0;
	printf("\n Output : ");
	while(arr[i]!='\0')
	{
		brr[j]=arr[i];
		j++;
		i++;
		if(arr[i]==' '||arr[i]=='\0')
		{
			j=j-1;
			while(j>=0)
			{
				printf("%c",brr[j]);
				j--;
			}
			printf(" ");
			memset(brr,'\0',20);
			j=0;
		}
	}
	printf("\n");
}

int main()
{
	char arr[30];
	printf("\n Enter your string : ");
	scanf("%[^'\n']s",arr);
	Reverse(arr);
	
}		